src/typescript/frontend/src/components/trade/DepositWithdrawModal/SelectMarketContent.tsx

```diff
@@ -1,116 +1,490 @@
+                      }
+
+                      // recognized
+                      if (header.id === "recognized") {
+                        if (
+                          selectedTab === 0 &&
+                          header.column.getFilterValue() == undefined
+                        ) {
+                          header.column.setFilterValue(true);
+                        }
+                        if (
+                          selectedTab === 1 &&
+                          header.column.getFilterValue() == true
+                        ) {
+                          header.column.setFilterValue(undefined);
+                        }
+                      }
+                      return (
+                        <th
+                          className={`${i === 0 ? "text-left" : ""} ${
+                            header.id === "recognized" ||
+                            (header.id === "24h_change" && "text-center")
+                          }
+          ${i === 0 ? TABLE_SPACING.paddingLeft : ""}
+          ${
+            i === headerGroup.headers.length - 1
+              ? TABLE_SPACING.paddingRight
+              : ""
+          } `}
+                          key={header.id}
+                        >
+                          {header.isPlaceholder
+                            ? null
+                            : flexRender(
+                                header.column.columnDef.header,
+                                header.getContext()
+                              )}
+                        </th>
+                      );
+                    })}
+                  </tr>
                 ))}
-              </tr>
-            ))
-          )}
-        </tbody>
-      </table>
+              </thead>
+              <tbody>
+                <tr>
+                  <td colSpan={7} className="">
+                    <div className="h-4"></div>
+                  </td>
+                </tr>
+                {isLoading || !data ? (
+                  <tr>
+                    <td colSpan={7}>
+                      <div className="flex h-[150px] flex-col items-center justify-center text-sm font-light uppercase text-neutral-500">
+                        Loading...
+                      </div>
+                    </td>
+                  </tr>
+                ) : data.length === 0 ? (
+                  <tr>
+                    <td colSpan={7}>
+                      <div className="flex h-[150px] flex-col items-center justify-center text-sm font-light uppercase text-neutral-500">
+                        No markets to show
+                      </div>
+                    </td>
+                  </tr>
+                ) : (
+                  table.getRowModel().rows.map((row) => (
+                    <tr
+                      className="h-24 min-w-[780px] cursor-pointer px-6 text-left font-roboto-mono text-sm text-white hover:outline hover:outline-1 hover:outline-neutral-600 [&>th]:font-light"
```

Comment:
The hover border and modal border blend into one another, so it looks kind of strange and unclear what market is being selected, especially with the last market on the list. Maybe we can come up with something better?

Reply:
good point, will bring up with @xorkiki

Reply:
ah just saw slack, nvm